### ISSUE TITLE
bug(MatChipList): Autocomplete fails when mat-chip-list is required

### ISSUE THREAD

**Issue by jhajkom** [angular/components#19391]
#### Reproduction

Use StackBlitz to reproduce your issue: https://stackblitz.com/edit/chips-autocomplete-required-failure?file=app/chips-autocomplete-example.ts

Steps to reproduce:
1. Fork the mat-chips autocomplete example from the Angular documentation
2. Move the form control to the mat-chip-list and add required attribute
3. Try using the auto complete 💥 

NOTE: Followed suggestion in the following issue to add the required field, but that example doesn't include autocomplete. With this the autocomplete breaks. https://github.com/angular/components/issues/14195
 
#### Expected Behavior

The autocomplete should filter results based on text entered. 

#### Actual Behavior

The autocomplete does not filter the results based on the text input from the user.

#### Environment

- Angular: 8.2.14
- CDK/Material: 8.2.3
- Browser(s): Chrome 81.0.4044.138
- Operating System (macOS): Mojave 10.14.6

**Comment by mmalerba** [comment]
Confirmed that this is still an issue, here's an updated stackblitz: https://stackblitz.com/edit/chips-autocomplete-required-failure-ejmtxd?file=material-module.

**Comment by mmalerba** [comment]
Actually, I think this may just be an issue with the repro code. Here's an example where it seems to work correctly https://stackblitz.com/edit/angular-n6tpca-k4imsc?file=app/chips-autocomplete-example.html

**Comment by jhajkom** [comment]
Hello @mmalerba thank you for your response, I believe this is still an issue. Using a mat-hint with a `mat-error` class is not the same as using a mat-error. The filed is also not responsive in terms of the validation. What I mean by that is that it does not turn red when there are errors. Your mat-error with text `boooe` does not show when the form is invalid.

In the example provided in the above description ([#14195](https://github.com/angular/components/issues/14195)) the recommendation was to move the form control to the `mat-chip-list` where the form validation works perfect. However, you then cannot use the autocomplete as it does not predict words based on what you type.

**Comment by mmalerba** [comment]
Sorry, is this more like what you were trying to do? https://stackblitz.com/edit/chips-autocomplete-required-failure-txn7nn?file=app/chips-autocomplete-example.ts

It does feel like it could work a little better out of the box, without me having to manually update the control value in a bunch of different listeners. I'll reopen this issue to track improving the out of the box experience

**Comment by jhajkom** [comment]
@mmalerba Yeah, your second example is exactly it. Thanks for taking the time to look into it and reopening the ticket. 🙂

**Comment by DDDarylll** [comment]
@mmalerba it will have a red border for 0.1 second when click an option, how can i solve this unexpected behavior
